crates/katana/storage/provider/src/providers/fork/backend.rs

```diff
@@ -242,6 +268,33 @@ where
             }
         }
     }
+
+    fn dedup_request<F>(
+        &mut self,
+        req_key: BackendRequestIdentifier,
+        sender: OneshotSender<BackendResponse>,
+        rpc_call_future: F,
+    ) where
+        F: FnOnce() -> BoxFuture<'static, BackendResponse>,
```

Comment:
is there a reason why this need to be a `FnOnce` and not just `BoxFuture<'static, BackendResponse>` ?

Reply:
Now that I think about it, not really, going to change it to just a BoxFuture.